dapr/proto/runtime/v1/dapr.proto

```diff
@@ -178,6 +178,9 @@ message PublishEventRequest {
 
   // The data which will be published to topic.
   bytes data = 3;
+
+  // The content type for the data (optional).
+  string dataContentType = 4;

```

Comment:
Please follow the existing convention. Member naming is snake case, that is what protobuf official guideline recommends. Then Protoc will generate the member based on language it could be camel case or snake case depending on the language.

Reply:
I will need to fix this. Thanks.